src/portable/nuvoton/nuc121/dcd_nuc121.c

```diff
@@ -229,6 +229,15 @@ void dcd_set_config(uint8_t rhport, uint8_t config_num)
 {
   (void) rhport;
   (void) config_num;
+
+  /* reset context of all non-control endpoints */
+
+  for (enum ep_enum ep_index = PERIPH_EP2; ep_index < PERIPH_MAX_EP; ep_index++)
```

Comment:
since we are removing `dcd_set_config()`, please remove the use of this API. Then we are ready to merge